install-pcf/vsphere/params.yml

```diff
@@ -76,6 +76,7 @@ deployment_nw_gateway:        # Deployment network Gateway
 deployment_nw_azs:            # Comma seperated list of AZ's to be associated with this network
 
 services_network_name: "SERVICES"
+service_nw:                   # Required select service network option in Ops man true or false
```

Comment:
It's better to use full words rather than compressed ones like `nw`. And since it's a boolean, perhaps something to that effect should be added, so: `is_service_network`